### PR TITLE
Hair 50 business phone

### PR DESCRIPTION
**Pull request by cynthiaeddy**
i updated the business filtered page and added a placeholder for the filters component, plus a heading for the page (hairstylists located in whatever city) to reflect the figma 'sketch':
Screen Shot 2021-11-19 at 5 13 33 PM

add phone number to business schema
Screen Shot 2021-11-19 at 4 58 56 PM

add phone number to business profile page
Screen Shot 2021-11-19 at 4 59 16 PM

phone number doesn't show up on individual business until book now is clicked
not clicked:
Screen Shot 2021-11-19 at 5 01 55 PM
clicked:
Screen Shot 2021-11-19 at 5 02 01 PM

review button doesn't show up on individual business unless logged in: (see preceding screenshots when user is logged in)
Screen Shot 2021-11-19 at 5 00 16 PM

### CODE REVIEW

**Comment by cynthiaeddy** [comment]
ps. phone number library does have a drop-down for as many countries as we want, I just set it to the us for now.

**Comment by cynthiaeddy** [comment]
also, phone number isn't 'styled' with dashed, parens

File: client/src/pages/stash/BusinessesFiltered copy.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
```
Comment (CodeMeKathy) [inline client/src/pages/stash/BusinessesFiltered copy.js]: Hi @cynthiaeddy! Is this duplicate file needed?

**Review by CodeMeKathy — APPROVED** [review]
Great work! Thank you for your efforts, @cynthiaeddy!

File: client/src/pages/stash/BusinessesFiltered copy.js (hunk 1 above)
Comment (cynthiaeddy) [reply client/src/pages/stash/BusinessesFiltered copy.js]: nope, deleted, will commit then merge

**Pull request merged**